contracts/SpokePool.sol

```diff
@@ -1232,28 +1297,11 @@ abstract contract SpokePool is
 
         // If leaf's amountToReturn is positive, then send L2 --> L1 message to bridge tokens back via
         // chain-specific bridging method.
-        if (relayerRefundLeaf.amountToReturn > 0) {
-            _bridgeTokensToHubPool(relayerRefundLeaf);
-
-            emit TokensBridged(
-                relayerRefundLeaf.amountToReturn,
-                relayerRefundLeaf.chainId,
-                relayerRefundLeaf.leafId,
-                relayerRefundLeaf.l2TokenAddress,
-                msg.sender
-            );
-        }
+        if (amountToReturn > 0) {
+            _bridgeTokensToHubPool(amountToReturn, l2TokenAddress);
 
-        emit ExecutedRelayerRefundRoot(
-            relayerRefundLeaf.amountToReturn,
-            relayerRefundLeaf.chainId,
-            relayerRefundLeaf.refundAmounts,
-            rootBundleId,
-            relayerRefundLeaf.leafId,
-            relayerRefundLeaf.l2TokenAddress,
-            relayerRefundLeaf.refundAddresses,
-            msg.sender
-        );
+            emit TokensBridged(amountToReturn, chainId(), leafId, l2TokenAddress, msg.sender);
```

Comment:
ooc, is it still relevant to emit `msg.sender` in this event? We're incidentally executing via Multicall3 so it doesn't really give any useful information at present. The same applies for the `...Executed` events.

Side note: We could make the bot a bit sharper so it avoids Multicall3 in cases where there is only one leaf to be executed, and in fact I think we only get a benefit from Multicall3 on the HubPool chain.

![tokensbridged](https://github.com/across-protocol/contracts-v2/assets/108695806/18d8f6bd-bfde-4c6b-983b-fccff534d3ac)

![relayerRefund](https://github.com/across-protocol/contracts-v2/assets/108695806/5548bc2b-aa99-4c10-8454-12a02e0caf80)

Reply:
its probably overkill yeah, `msg.sender` is not useful for the most part because we execute everything ourselves but what about a world where there are multiple executors? This would make it easier to track perhaps?

Reply:
5770b3f821959a0c97221f73764ec380e14a7c43

I don't think this neccessarily breaks bots but we should remove from sdk-v2/interfaces/SpokePool

https://github.com/across-protocol/sdk-v2/blob/master/src/interfaces/SpokePool.ts#L231

Reply:
> Side note: We could make the bot a bit sharper so it avoids Multicall3 in cases where there is only one leaf to be executed, and in fact I think we only get a benefit from Multicall3 on the HubPool chain.

Also, don't we handle that here?

https://github.com/across-protocol/relayer-v2/blob/master/src/clients/MultiCallerClient.ts#L407